src/object_properties.cpp

```diff
@@ -166,4 +167,5 @@ void ObjectProperties::deSerialize(std::istream &is)
 	eye_height = readF32(is);
 	zoom_fov = readF32(is);
 	use_texture_alpha = readU8(is);
+	damage_texture_modifier = deSerializeString(is);
```

Comment:
needs backwards compatibility

Reply:
sfan5: how should I implement that? checking for `is_open` ?